application/routes.py

```diff
@@ -2,12 +2,19 @@
 from flask import render_template
 import random
 
+## Use flask's server-side rendering to create a page from templates/index.html
+## The @app.route decoration tells flask to return this content for both http://<root> and http://<root>/index
 @app.route('/')
 @app.route('/index')
 def index():
     return render_template('index.html',
             title='Platonic-Dash Demo')
 
+## A second page rendered by Flask rather than Dash.
+## Uses Flask's templating to choose a random string from an array of 3 strings
+## and insert the string into the page before serving the page to the user.
+## @app.metrics allows adding Prometheus instrumentation, in this case,
+## a counter of how many times the page is loaded.
```

Comment:
Yes, __init__.py exists to hold code to be executed on module initialization. I've used this pattern because I have seen in reflected in other Flask projects and tutorials. My goal with this project is to make Dash development more like standard Flask.

Reply:
Sounds reasonable. I would add `# Import Flask routes, which modifies the app variable (https://flask.palletsprojects.com/en/1.1.x/appcontext/)`